packages/siwe/lib/client.test.ts

```diff
@@ -42,29 +42,24 @@ describe(`Message Generation`, () => {
   );
 });
 
-describe(`Message verification without suppressExceptions`, () => {
+describe(`Message verification`, () => {
   test.concurrent.each(Object.entries(verificationPositive))(
     'Verificates message successfully: %s',
     async (_, test_fields: any) => {
       const msg = new SiweMessage(test_fields);
+      const params = {
+        signature: test_fields.signature,
+        time: ((test_fields as any).time || test_fields.issuedAt),
```

Comment:
I don't know if any test is using `time` yet, but maybe it would avoid any potential confusion but putting it under a field called something like `test_params`

Reply:
I wasn't exactly sure the relevance of `time` is it only used for tests? I was under the impression it was part of the `SiweMessage`

Reply:
I'm certain it's only for testing purposes, to mock the system's clock